gpflow/kullback_leiblers.py

```diff
@@ -80,26 +83,49 @@ def gauss_kl(q_mu, q_sqrt, K=None):
         trace = tf.reduce_sum(tf.square(Lq))
     else:
         if diag:
```

Comment:
Can you get rid of `if if else ifelse else if` :)?

Reply:
Well, there is no easy way to do that.
There are different treatment for the factors white x batch x diag...
I can do some inline ifs if that helps ,,,, like if x then y else z.